app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -178,6 +179,16 @@ interface SessionControlController {
      * @see [ExperimentCardInteractor.onCloseExperimentCardClicked]
      */
     fun handleCloseExperimentCard()
+
+    /**
+     * @see [RecentTabInteractor.onRecentTabClicked]
+     */
+    fun handleRecentTabClicked(tabId: String)
```

Comment:
We spoke about this offline, but leaving a note for myself to remember: Let's move this to a separate controller/interactor because we know that the "recent tabs" section is going to grow and become bigger over time.

Reply:
![external-content duckduckgo com](https://user-images.githubusercontent.com/1370580/121579200-81ffd980-c9f9-11eb-9062-1dadfdbdd90b.gif)